cfme/tests/cli/test_appliance_console.py

```diff
@@ -844,7 +843,16 @@ def test_appliance_console_evm_start():
             4. Confirm EVM service is being started.
             5. Confirm replication is working correctly
     """
-    pass
+    appliance.evmserverd.stop()
+    wait_for(lambda: appliance.is_web_ui_running(), delay=30, num_sec=10, fail_condition=True)
+
+    # Actual testcase starting from here
+    start_evm_command = ("ap", RETURN, "17", "Y", RETURN, RETURN, "21")
+    appliance.appliance_console.run_commands(start_evm_command, timeout=300)
+    wait_for(lambda: appliance.is_web_ui_running(), delay=30, num_sec=10)
+    logged_in_page = appliance.server.login()
+    assert logged_in_page.is_displayed, "UI is not working after starting the EVM service."
+    logged_in_page.logout()
 
```

Comment:
@dgaikwad 
If assertion gets failed then this line(```logged_in_page.logout()```) will not execute. So is it okay if we add this in ```request.addfinalizer``` above the assertion. Thus it will act as teardown for this test case.